code/modules/mob/living/carbon/human/species.dm

```diff
@@ -329,6 +332,7 @@
 	brute_mod = 1.05
 	burn_mod = 1.05
 	slowdown = 1.05
+	joinable_roundstart = FALSE
```

Comment:
Why you cant use vat grown as your species? I just saw many peoples enjoyed having vat grown chars.

Reply:
Vat-growns and vatborns are entirely two different things.

Vat-growns are basically cloned guys during the round, and they can take slightly less damage.
Vatborns are also cloned guys, but they have matured well enough to survive like a human, performing everything like a human could.